backend/cmd/start.go

```diff
@@ -182,18 +185,6 @@ func newStartCommand() *cobra.Command {
 					TrustedCAFile:      trustedCAFile,
 					InsecureSkipVerify: insecureSkipTLSVerify,
 				}
-			} else if certFile != "" || keyFile != "" || trustedCAFile != "" {
-				emptyFlags := []string{}
-				if certFile == "" {
-					emptyFlags = append(emptyFlags, flagCertFile)
-				}
-				if keyFile == "" {
-					emptyFlags = append(emptyFlags, flagKeyFile)
-				}
-
-				if len(emptyFlags) > 0 {
-					return fmt.Errorf("missing the following cert flags: %s", emptyFlags)
-				}
```

Comment:
I think we should keep a form of that branch that checks that _both_ `certFile` and `keyFile` are present at the same time because, unless I'm mistaken, providing one without the other doesn't make sense.

ie: keep that branch, drop the `trustedCAFile != ""` condition, reword the error message to say "you used X without Y" (or the opposite) accordingly.

Reply:
well technically according to RFC, one is allowed to include both a certificate and private key in the same pem file. And Go supports loading both from same pem file as well. However, in practice, I have never seen people do this lol. OK I will add back the friendly error if one is missing :)